tools/icon_cutter/check.py

```diff
@@ -72,7 +72,14 @@ def hash_file(path):
 pass_count = 0
 fail_count = 0
 output_hash = {}
-for cutter_template in glob.glob("..\\..\\icons\\**\*.toml", recursive = True):
+path_to_us = os.path.realpath(os.path.dirname(__file__))
+print(path_to_us)
```

Comment:
what's the point of the `print()` here? is it code left over from debugging? if there is a purpose behind it then disregard

Reply:
ah debugging leftover